go/vt/vtgate/planbuilder/show.go

```diff
@@ -275,6 +275,22 @@ func buildPlanWithDB(show *sqlparser.ShowBasic, vschema ContextVSchema) (engine.
 
 }
 
+func buildShowStatusPlan(show *sqlparser.ShowBasic, vschema ContextVSchema) (engine.Primitive, error) {
+	var ks *vindexes.Keyspace
+	var err error
+
+	if ks, err = vschema.AnyKeyspace(); err != nil {
+		return nil, err
+	}
+
+	return &engine.Send{
+		Keyspace:          ks,
+		TargetDestination: key.DestinationAnyShard{},
+		Query:             sqlparser.String(show),
+		SingleShardOnly:   true,
+	}, nil
+}
+
```

Comment:
this looks equivalent to `buildSendAnywherePlan`, it can be changed to use that function.

Reply:
yup. its exactly the same :)